discord-scripts/fix-twitter-embed.ts

```diff
@@ -0,0 +1,59 @@
+import { PartialMessage, Client, Message, EmbedBuilder } from "discord.js"
+
+async function workingTwitterEmbeds(
+  message: Message<boolean> | PartialMessage,
+) {
+  if (message.author === null || message.author === undefined) {
+    return
+  }
+
+  if (!message.author.bot) {
```

Comment:
I like de-nesting this:

```ts
	if (message.author.bot) {
		return
	}

	const content = message.content ?? ""
	if (content.match(/(https:\/\/(x|twitter).com\/[a-zA-Z0-9%/+]+)/)) {
		const allLinks = Array.from(
		...
```

Reply:
What I'd like to see here is a linter rule that enforces one way or the other. Otherwise I don't think we can push either approach scalably. 

Going to merge and we can figure the rule out separately.